src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-migrate/rgw-multisite-migrate.component.ts

```diff
@@ -174,11 +175,10 @@ export class RgwMultisiteMigrateComponent implements OnInit {
     this.zone = new RgwZone();
     this.zone.name = values['zoneName'];
     this.zone.endpoints = values['zone_endpoints'];
-    this.zone.system_key = new SystemKey();
-    this.zone.system_key.access_key = values['access_key'];
-    this.zone.system_key.secret_key = values['secret_key'];
-    this.rgwMultisiteService.migrate(this.realm, this.zonegroup, this.zone).subscribe(
+    const username = values['username'];
+    this.rgwMultisiteService.migrate(this.realm, this.zonegroup, this.zone, username).subscribe(
```

Comment:
if `username` has no other purpose, let's use that directly in the function.
```suggestion
    this.rgwMultisiteService.migrate(this.realm, this.zonegroup, this.zone, values['username']).subscribe(
```